flutter/lib/src/sentry_privacy_options.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:meta/meta.dart';
+
+import '../sentry_flutter.dart';
+import 'screenshot/masking_config.dart';
+import 'screenshot/widget_filter.dart';
+
+/// Configuration of the experimental privacy feature.
+class SentryPrivacyOptions {
+  /// Mask all text content. Draws a rectangle of text bounds with text color
+  /// on top. Currently, only [Text] and [EditableText] Widgets are redacted.
```

Comment:
```suggestion
  /// on top. Currently, only [Text] and [EditableText] Widgets are masked.
```

let's keep the naming consistent